components/datetime/src/time_zone.rs

```diff
@@ -165,17 +164,20 @@ impl<'data> TimeZoneFormat<'data> {
             .take_payload()?;
 
         let mut time_zone_format = Self {
-            patterns,
+            patterns: Some(patterns),
             zone_formats,
             exemplar_cities: None,
             mz_generic_long: None,
             mz_generic_short: None,
             mz_specific_long: None,
             mz_specific_short: None,
+            config: None,
```

Comment:
> nitpick

Same here with the grouping of `config` and `patterns` (though it makes no difference to the compiler).

```rust
let mut time_zone_format = Self {
    config: None,
    patterns: Some(patterns),
    zone_formats,
    exemplar_cities: None,
    mz_generic_long: None,
    mz_generic_short: None,
    mz_specific_long: None,
    mz_specific_short: None,
};
```